datus-hive/tests/integration/conftest.py

```
@@ -49,9 +54,181 @@ def connector(config: HiveConfig) -> Generator[HiveConnector, None, None]:
     except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)